### PR TITLE
SAK-30025 Add downloadable PA System popup templates

### PR DESCRIPTION
**Pull request by marktriggs**

### CODE REVIEW

File: pasystem/pasystem-tool/tool/pom.xml [hunk 1]
```diff
@@ -71,4 +71,44 @@
...
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-war-plugin</artifactId>
+        <version>2.4</version>
```
Comment (buckett) [inline pasystem/pasystem-tool/tool/pom.xml]: Why are you downgrading the war plugin here (we're generally using 2.6)? And I think the configurations you're supplying is the default anyway. 

https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#failOnMissingWebXml

**Comment by buckett** [comment]
Otherwise looks good to merge.

**Comment by marktriggs** [comment]
Thanks Matthew!  You're right that I didn't need to set that version or the configuration, so I've let that default now.  I only list `maven-war-plugin` at all to ensure that it runs after the template .zip file has been produced (since I need that .zip file to be bundled into the final .war).

**Comment by buckett** [comment]
Ah I see, I didn't twig that's why you'd included it. To me it makes more sense to just put the assembly plugin in `process-resources` phase that that's much closer to what it's doing in this case. Then you don't need the WAR definition at all.

**Comment by buckett** [comment]
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html is a description of the maven lifecycle

**Comment by marktriggs** [comment]
Ah!  That does seem like a better idea.  I've just amended the commit and dropped the `maven-war-plugin` bit:

https://github.com/sakaiproject/sakai/commit/3b94bd9d7106d94bf1b92f4b48e59d0296dce886#diff-c8cdd9a1326c2d391750496a8d6e09f7R74

Thanks for the suggestion :)

**Comment by buckett** [comment]
Magic, thank.

**Pull request merged**